build-tools/packages/build-cli/src/lib/bump.ts

```diff
@@ -259,11 +266,11 @@ export async function bumpReleaseGroup(
 	let range: string;
 	if (scheme === "internal" || scheme === "internalPrerelease") {
 		range =
-			exactDependencyType === ""
+			interdependencyType === ""
 				? translatedVersion.version
```

Comment:
what does `interdependencyType === ""` mean?

Reply:
I added a comment to clarify.